### DISCUSSION TITLE
Disable Wi-Fi

### DISCUSSION THREAD

**Discussion by kiquenet** [farag2/Sophia-Script-for-Windows#153] [Q&A] [answered]
If you use LAPTOP using network RJ-45, not Wi-Fi

Get-NetAdapter -Name Wi-Fi | Disable-NetAdapter -Confirm:$false
Get-NetAdapter -Name Wi-Fi | Enable-NetAdapter

Detect "Laptop" using Powershell ?

**Comment by farag2** [comment, marked as answer]
Its possible.

```powershell
(Get-CimInstance -ClassName CIM_ComputerSystem).PCSystemType
```

For laptops it will be `2`. For desktops `1`. https://docs.microsoft.com/ru-ru/windows/win32/cimwin32prov/win32-computersystem
